src/app/controllers/CalendarController.js

```diff
@@ -0,0 +1,62 @@
+import { celebrate, Segments, Joi } from 'celebrate';
+import CalendarEvent from '../schemas/CalendarEvent';
+
+class CalendarController {
+	async createEvent(req, res) {
+		const result = await CalendarEvent.create(req.body);
+
+		return res.json(result);
+	}
+
+	/**
+	 * Return upcoming and unfinished events
```

Comment:
don't you think it might be better to make the function name more descriptive instead of using comments? For example, using `getAllEvents` here and `getFinishedEvents`, `getUpcomingEvents`...